ntc_templates/templates/cisco_xr_admin_show_inventory.textfsm

```diff
@@ -1,15 +1,15 @@
 Value Required NAME (.*?)
-Value DESCR (.*)
+Value DESCR (.*?)
```

Comment:
I don't think this is needed.

Reply:
This is needed as it will catch the whitespace for the classic or virtual XR output. 

e.g. notice the whitespace in the below 'descr' values. 

```
[{'descr': 'Cisco IOS-XRv 9000 Centralized Virtual Router               ',
  'name': 'Rack 0',
  'pid': 'R-IOSXRV9000-CC',
  'sn': 'FOC1234PA56',
  'vid': 'V01'},
 {'descr': 'Cisco IOS-XRv 9000 Centralized Line Card                    ',
  'name': '0/0',
  'pid': 'R-IOSXRV9000-LC-C',
  'sn': 'FOC4321PA6D',
  'vid': 'V01'},
 {'descr': 'Cisco IOS-XRv 9000 Centralized Route Processor              ',
  'name': '0/RP0',
  'pid': 'R-IOSXRV9000-RP-C',
  'sn': 'FOC7085NP2K',
  'vid': 'V01'}]
  ```
  
  Please also refer to my script, I created for to test this

Reply:
From my testing `Value DESCR (.*?)` does seem to be necessary otherwise the whitespace is matched because of the current DESCR regex.

So far other alternatives I've tried then there's a trailing space since the regexes are greedy (by default).
Even simple `Value DESCR ((\S+\s)+)` has a trailing space.

Seeing as how the template has `Value Required NAME (.*?)` since June 2023, this may be fine as there's a precedent.